src/collection.js

```diff
@@ -271,6 +271,28 @@ class Collection {
     return this.items.join(separator)
   }
 
+  /**
+   * Returns the last item in the collection
+   * that satisfies the `callback` testing
+   * function, `undefined` otherwise.
+   *
+   * @param {Function} callback
+   *
+   * @returns {*} the found value
+   */
+  async last (callback) {
+    if (!callback) {
+      return this.items[this.items.length - 1]
+    }
+
+    if (typeof callback === 'function') {
+      const mapped = await this.filter(callback)
+      return mapped[mapped.length - 1]

```

Comment:
Can you please add an empty line before the return statement?

Reply:
Do you mean empty line before every return statement?